### ISSUE TITLE
Why is the structure so deep?

### ISSUE THREAD

**Issue by rossmeissl** [faradayio/cage#30]
@emk I feel like a Cage repo could simply be:

```
_config.yml
common.env
frontend.yml
overrides/
  development/
    common.env
```

Why all the fuss with the `config/` dir and the demotion of the `pod.yml` files to a `pods/` dir?

**Comment by emk** [comment]
Tl;dr: The structure is deep because as the project grows, things get a lot messier, and because I like relatively clean top-level directories.

- The `config` directory potentially also contains `config/secrets.yml`, `config/vault.yml`, `config/sources.yml`, `config/hooks/pull.d/`, etc. And it will grow as cage gains new plugins and as projects get bigger.
- Most projects will also have top-level `data` and `src` directories (and probably `scripts`).
- The `pods` directory is separate because it might easily contain a dozen pods (including task pods), some with `metadata.yml` for now. Dumping all this in the top-level directory makes projects look ugly in a GitHub overhead view, and might discourage _existing_ projects from adopting cage. But if we keep the clutter in `pods`, then existing projects can just throw in a `pods` and `config` directory to support cage, without it taking over their top level.

**Comment by emk** [comment]
As a followup to that last bullet, you should actually be able to write `build: ".."` in a pod now, and keep your main services's source in the root directory. Not that I recommend this. But it might make sense for an existing project switching to cage.

**Comment by rossmeissl** [comment]
Great, makes sense
